runtime/vm/JFRChunkWriter.cpp

```diff
@@ -144,11 +161,14 @@ VM_JFRChunkWriter::writeCheckpointEventHeader(CheckpointTypeMask typeMask, U_32
 	_bufferWriter->writeLEB128((U_64)0);
 
 	/* reserve delta offset to next checkpoint event */
-	_previousCheckpointDelta = _bufferWriter->getCursor();
-	_bufferWriter->writeLEB128PaddedU64(0);
+	_previousCheckpointDelta = reserveEventSize();
 
 	if (_debug) {
-		j9tty_printf(PORTLIB, "next pointer=%d val=%d\n",(int) _bufferWriter->getFileOffsetFromStart(_previousCheckpointDelta), (int) VM_BufferWriter::convertFromLEB128ToU64(_previousCheckpointDelta));
+		j9tty_printf(
+				PORTLIB,
+				"next pointer=%llX val=%llX\n",
```

Comment:
My comment was only with respect to the first parameter `next pointer=%llX` (which should be `next pointer=0x%llX` (with the leading `0x`) for consistency.

The second parameter was `val=%d`; I was expecting that to become `val=%llu`, but if Tobi prefers hex, it should also have the `0x` prefix.

Reply:
Ok I changed the second parameter back to `%llu`.